doc/classes/GraphNode.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="GraphNode" inherits="Container" category="Core" version="3.2">
 	<brief_description>
-		A GraphNode is a container with several input and output slots allowing connections between GraphNodes. Slots can have different, incompatible types.
+		A GraphNode is a container with potential several input and output slots allowing connections between GraphNodes. Slots can have different, incompatible types.
```

Comment:
typo potential -> potentially

Reply:
Thanks, I fixed that. Anything else is welcomed. GraphNode is a complex node and hard to grasp without watching tutorial so I want this description to be as detailed and easy to understand as possible.